io.github.fastrizwaan.WineZGUI.yml

```diff
@@ -0,0 +1,151 @@
+id: io.github.fastrizwaan.WineZGUI
+branch: '23.08'
+version: '0.97.12'
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+base: org.winehq.Wine
+base-version: stable-23.08
+command: winezgui
+#rename-icon: winezgui
+#rename-desktop-file: winezgui.desktop
+finish-args:
+  - --allow=devel # For Wine crash handling
+  - --allow=multiarch
+  - --device=all
+  - --env=GST_PLUGIN_SYSTEM_PATH=/app/lib32/gstreamer-1.0:/app/lib/gstreamer-1.0:/usr/lib/i386-linux-gnu/gstreamer-1.0:/usr/lib/x86_64-linux-gnu/gstreamer-1.0
+  - --env=WINEDLLPATH=/app/dlls/lib32:/app/dlls/lib:/app/lib32/wine/wined3d:/app/lib/wine/wined3d
+    # Need to link .desktop files
+  - --filesystem=~/.local/share/applications:create
+  - --env=PATH=/app/bin:/usr/bin
+    # Desktop shortcut runs wine script from host
+  #- --talk-name=org.freedesktop.Flatpak
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --socket=x11
```

Comment:
either it support wayland and it's `fallback-x11` or it doesn't and then there is no wayland socket.